### DISCUSSION TITLE
Practica HTML del live 14-10-2022

### DISCUSSION THREAD

**Discussion by Cesar-AVF** [Launch-X-Latam/MisionFrontEnd#224] [Show and tell]
Hola colegas! Es la primer practica que realizo, se las comparto esperando me puedan retroalimentar, ya que considero el código no es muy limpio, por los estilos que intente hacer.

Saludos a todos!

https://github.com/Cesar-AVF/Practica-14-10-2022-Taquer-a-

**Comment by DaveOval** [comment]
bro, te recomiendo darle un orden a tus archivos ejemplo tener todo el proyecto en una carpeta, después dentro de esa carpeta tener el html principal llamándolo "index.html" y tener separado por carpetas cada tipo de archivos por ejemplo: las imágenes en una carpeta llamada "img", los estilos en una carpeta llamada "estilos" y así por cada tipo de archivos
